lxd/images.go

```diff
@@ -1573,7 +1573,31 @@ func imageDelete(d *Daemon, r *http.Request) response.Response {
 		}
 
 		for _, pool := range pools {
-			err := doDeleteImageFromPool(d.State(), imgInfo.Fingerprint, pool)
+			isRemote := false
```

Comment:
Wouldn't an easier approach in this case be to only perform the delete of remote storage volumes if `!isClusterNotification`?

This would make whatever server is handling the delete operation perform the deletion both for its local pools and for the remote pools while the rest just handle their local pools.

Reply:
That'll do it.